spec.bs

```diff
@@ -666,6 +680,9 @@ scope</dfn> given a [=pre-specified report parameters=] |params| and a
     filtering ID max bytes=].
```

Comment:
We should probably also update "determine if a report should be sent deterministically"

(Random location as it won't let me comment on lines too far away)